src/js/config.js

```diff
@@ -6,7 +6,7 @@ const appConfig = {};
 const versions = {};
 
 function fetchConfig(success) {
-  fetch('/appconfig.json', { data: { _: _NOW_ } })
+  fetch(`/appconfig.json?${ new URLSearchParams({ _: _NOW_ }) }`)
```

Comment:
`_NOW_` is the unix timestamp at the time of the FE asset build.  It's not variable.